softlayer-cpi.html.md.erb

```diff
@@ -169,5 +185,7 @@ compilation:
 
 The ``Bosh_ip`` property specified under ``cloud_properties`` is used for Softlayer CPI to differentiate the director and common vms. The one with cloud_property ``Bosh_ip`` is a common vm. The one without ``Bosh_ip`` is the director.
 
+Please notice that when the VM hostname length is exactly 64, the deployment is failing due to ssh problem. This is Softlayer’s limitation which can’t be fixed in a short term. We have a work around in the CPI that when the hostname with 64 characters is identified, a padding "-1" is appended to make it longer than 64.
```

Comment:
The SL ticket can only been read by the same account which opened it. I think it doesn't make sense to paste SL ticket link here as others can't read it.